modules/Bio/EnsEMBL/Variation/Phenotype.pm

```diff
@@ -118,6 +119,23 @@ sub dbID {
   return $self->{'dbID'};
 }
 
+=head2 stable_id
+
+  Example    : $name = $obj->stable_id()
```

Comment:
Suggest using $stable_id = $obj->stable_id() to avoid confusion with method name example
 $name = $obj->name()